### ISSUE TITLE
Post from Rstudio to Github

### ISSUE THREAD

**Issue by palmierieugenio** [aurora-mareviv/aurora-mareviv.github.io#2]
How can you make a new post in Rstudio and publish it on Github?

I've been able to do it locally, but I don't know how to make the change on Github (I am not an expert with Github), so the only way I have to do it right now is to deleting and recreating the site everytime from zero (deleting the github folder).

Is there a way to link my local site directory with the github repository, so that I can make the changes automatically?

Or at least a way to substitute "manually" the content of the github repository with the one (updated) of my folder? (or other more practical ways to use posts/contents)

**Comment by palmierieugenio** [comment]
This seems a way to do it:

git push origin dev --force

I don't know if it's the best one.

Source:
https://stackoverflow.com/questions/8386996/how-can-i-choose-to-overwrite-remote-repository-with-local-commits

**Comment by aurora-mareviv** [comment]
I have just got into the issue. You don't need to do this to get your new post updated.  
I have done a brief (temporary) tutorial which I will merge into the older one. See the info [here](https://aurora-mareviv.github.io/2017/08/r-blogdown-setup-in-gihub/).   
I know there is a procedure to create a new post using the interface in RStudio but if it doesn't work for you for some reason, you always can create a new *.Rmd* by hand and place it into the *contents/post* folder, with an appropriate YAML header.   
The `git` commands are the same.

**Comment by palmierieugenio** [comment]
Thank you very much, it was very helpful. :)

Another problem now is that the links of the images breaks, so I have to put a direct link manually.

Maybe changing from:
baseurl = "."
publishDir = "."

to:
baseurl = "/./"
publishDir = "./"

could help, I don't know.

I hope the new R/Rstudio/blogdown versions will simplify these kinds of problems, because it seems potentially a very good tool ^^

**Comment by palmierieugenio** [comment]
Now it works with:
baseurl = "/./"
publishDir = "./"
:)

**Comment by aurora-mareviv** [comment]
Yes! I arrived to the same conclusion. I modified it in the `config.toml` file but forgot to update the tutorial. Now it's fixed.   
Regarding the path inconsistency in/between themes, I don't know if it has an easy solution (if you want to avoid modifying them from Hugo itself). I hope that this will improve over time, but the problem seems to be in the different themes more than in `blogdown`.  See this [reference](https://bookdown.org/yihui/blogdown/other-themes.html) in the blogdown manual.
